spec/features/projects/edit.html.erb_spec.rb

```diff
@@ -0,0 +1,125 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe 'projects/edit.html.erb', type: :feature do
+  let(:user) { FactoryBot.create :user }
+  let(:employee) { FactoryBot.create :employee }
+  let(:admin) { FactoryBot.create :admin }
+
+  let(:project) { FactoryBot.create :project, responsible_users: [employee, admin] }
+  let(:project_name) { 'Name' }
+  let(:project_description) { 'Description' }
+
+  def submit_button
+    page.find(id: 'submitProjectButton')
+  end
+
+  before do
+    user
+    employee
+    admin
+    project
+  end
+
+  context 'when the user is not an responsible user' do
+    before do
+      sign_in user
```

Comment:
One more idea if it makes sense to you, In the outer before do do:

sign_in current_user
visit edit_project_path(project)

and in the 2 contexts set the current user appropriately